cpp/src/barretenberg/solidity_helpers/circuits/add_2_circuit.hpp

```diff
@@ -0,0 +1,32 @@
+#include "barretenberg/stdlib/primitives/field/field.hpp"
+#include "barretenberg/stdlib/primitives/witness/witness.hpp"
+#include "barretenberg/stdlib/primitives/uint/uint.hpp"
+#include "barretenberg/stdlib/primitives/bool/bool.hpp"
+
+// #include <stdlib/primitives/field/field.hpp>
```

Comment:
removable

Reply:
Addressed in https://github.com/AztecProtocol/barretenberg/pull/363/commits/ca5f638df7a462a6b102d6a7e86a94759a95d335